### PR TITLE
Added a new Company Named Basis Point Design

### PR DESCRIPTION
**Pull request by AnuragNishad12**
Just Add a new Company In .md files called "Basis Point Design Ltd" 

Company Website :-
https://basispoint.design/

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Name | Website | Region
 [Balsamiq](/company-profiles/balsamiq.md) | https://balsamiq.com/ | Worldwide
 [Bandcamp](/company-profiles/bandcamp.md) | https://bandcamp.com/ | Worldwide
 [BandLab](/company-profiles/bandlab.md) | https://bandlab.com/ | Worldwide
+[BasisPointDesign](/company-profiles/basisPointDesignLtd.md) | https://basispoint.design/ | Portugal, South Africa
 [Bandzoogle](/company-profiles/bandzoogle.md) | https://bandzoogle.com/ | Worldwide
 [Baremetrics](/company-profiles/baremetrics.md) | https://baremetrics.com/ | Worldwide
 [Basecamp](/company-profiles/basecamp.md) | https://basecamp.com/ | Worldwide
```
Comment (KieranRobson) [inline README.md:83]: Incorrect Order.

```suggestion
[Bandcamp](/company-profiles/bandcamp.md) | https://bandcamp.com/ | Worldwide
[BandLab](/company-profiles/bandlab.md) | https://bandlab.com/ | Worldwide
[Bandzoogle](/company-profiles/bandzoogle.md) | https://bandzoogle.com/ | Worldwide
[Baremetrics](/company-profiles/baremetrics.md) | https://baremetrics.com/ | Worldwide
[Basecamp](/company-profiles/basecamp.md) | https://basecamp.com/ | Worldwide
[BasisPointDesign](/company-profiles/basisPointDesignLtd.md) | https://basispoint.design/ | Portugal, South Africa
```

File: company-profiles/basisPointDesignLtd.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# basisPointDesignLtd
```
Comment (KieranRobson) [inline company-profiles/basisPointDesignLtd.md:1]: Filename must match the Title.

```suggestion
# basisPointDesignltd
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple of changes to be made.

File: company-profiles/basisPointDesignLtd.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# basisPointDesignltd
```
Comment (KieranRobson) [inline company-profiles/basisPointDesignLtd.md:1]:
```suggestion
# basispointdesignltd
```

File: company-profiles/basisPointDesignLtd.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# basispointdesignltd
```
Comment (dougaitken) [inline company-profiles/basisPointDesignLtd.md:1]: The company name and file name should match the name listed in the readme.md file. This company is listed as `BasisPointDesign` so the file name should be `basispointdesign.md` and the title here should be `# BasisPointDesign`

File: company-profiles/basisPointDesignLtd.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Basis Point Design is a brand, web and marketing communications specialist that services emerging and established investment managers across the globe.
+
+## Company size
+
+11-50 employees
+
+## Remote status
+
+Branding, Graphic Design, Marketing Presentations, Website Design, Website Development, Marketing Communications, Investor Communications, Pitch Deck Design, Digital Design, Investor Portal Design and Build, Marketing Emailers, Video Production, Animation, Infographics, UI UX Design, Infographics, 3D Motion, Marketing Videos, Audio Pitch Decks, Fact Sheet Design, and Investor Reporting
```
Comment (dougaitken) [inline company-profiles/basisPointDesignLtd.md:13]: This copy has nothing to do with the remote status. As mentioned [in the example](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md#remote-status), this should related to the remote culture of the company.

File: company-profiles/basisPointDesignLtd.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+## Region
+
+England
```
Comment (dougaitken) [inline company-profiles/basisPointDesignLtd.md:17]: The region is where remote employees are accepted - https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md#region - this doesn't match the region in the readme file.

File: company-profiles/basisPointDesignLtd.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+## Company technologies
+
+WordPress, JavaScript, CSS, SCSS, HTML,Next.js, Typescript, Node.js, Vue.js, React.js, Gatsby.js,PHP, Laravel, MySql
+
+## How to apply
+
+Visit https://basispoint.design/
```
Comment (dougaitken) [inline company-profiles/basisPointDesignLtd.md:25]: I don't see any mention about open careers or remote employment. Can you add context, please?

**Review by dougaitken — CHANGES_REQUESTED** [review]
Please review the comments and fix before this will be merged, otherwise it will be closed without merging as invalid

**Pull request closed without merging**